ui/src/components/WorkersTable/index.jsx

```diff
@@ -328,15 +328,22 @@ export default class WorkersTable extends Component {
                 {providerId !== NULL_PROVIDER && state !== 'stopping' && (
                   <Button
                     requiresAuth
-                    disabled={['stopping', 'stopped'].includes(state)}
+                    disabled={
+                      ['stopping', 'stopped'].includes(state) ||
```

Comment:
`stopping` will not reach this point, as it is being filtered out 4 lines above this

Reply:
Good catch, thank you! I've decided that for here, I'll just not even show the terminate button to the users if the state is `stopping` or `stopped`.